packages/fetchai/skills/buyer_aw2/README.md

```diff
@@ -0,0 +1,24 @@
+# Car Park Client
+
+## Description
+
+This skill purchases information from other agents as specified in its configuration.
+
+This skill finds an agent on the SOEF which sells the sought data in a vicinity, requests this data, negotiates the price, pays the proposed amount if agreement is reach, and receives the data bought.
```

Comment:
sought might be too complex for some.

`This skill finds an agent on the SOEF which sells the data it needs in a vicinity,...`